lighthouse-core/test/lib/icons-test.js

```diff
@@ -278,7 +278,9 @@ describe('Icons helper', () => {
       assert.equal(icons.pngSizedAtLeast(192, manifest.value).length, 0);
     });
 
-    it('fails with an icon that has a png typehint but is not png', () => {
+    it('succeeds with an icon that has a png typehint but is not png', () => {
+      // We will believe your typehints until such time that we can fetch the image and decode it
+      // TODO: fetch images and decode them to check real filetype like in Chrome
```

Comment:
maybe
`// Note: we will believe your typehints until we can fetch the image and decode it. See https://github.com/GoogleChrome/lighthouse/issues/789`

(TODO is preferably not in the test)